api/envoy/config/filter/network/redis_proxy/v2/redis_proxy.proto

```diff
@@ -24,7 +24,7 @@ option (udpa.annotations.file_status).package_version_status = FROZEN;
 // Redis Proxy :ref:`configuration overview <config_network_filters_redis_proxy>`.
 // [#extension: envoy.filters.network.redis_proxy]
 
-// [#next-free-field: 7]
+// [#next-free-field: 8]
```

Comment:
Can you add this to the v3 API? We've frozen the v2 API. Thanks.

Reply:
Will do. To be clear, this means removing changes from `v2/redis_proxy.proto`, but keeping them in the v3 file of the same name?